docker/Dockerfile

```diff
@@ -33,17 +33,11 @@ RUN pyenv install -v ${PYTHON_VERSION}
 RUN pyenv global ${PYTHON_VERSION}
 
 # Install TensorRT + dependencies
-RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-ubuntu2204.pin
-RUN mv cuda-ubuntu2204.pin /etc/apt/preferences.d/cuda-repository-pin-600
-RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/7fa2af80.pub
-RUN add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/ /"
-RUN apt-get update
-
 RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/3bf863cc.pub
 RUN add-apt-repository "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/ /"
 RUN apt-get update
-
-RUN apt-get install -y libnvinfer8=${TENSORRT_VERSION}.* libnvinfer-plugin8=${TENSORRT_VERSION}.* libnvinfer-dev=${TENSORRT_VERSION}.* libnvinfer-plugin-dev=${TENSORRT_VERSION}.* libnvonnxparsers8=${TENSORRT_VERSION}.* libnvonnxparsers-dev=${TENSORRT_VERSION}.* libnvparsers8=${TENSORRT_VERSION}.*  libnvparsers-dev=${TENSORRT_VERSION}.* libnvinfer-headers-dev=${TENSORRT_VERSION}.* libnvinfer-headers-plugin-dev=${TENSORRT_VERSION}.*
+RUN python3 -m pip install numpy
+RUN apt-get install -y python3-libnvinfer-dev
```

Comment:
@lanluo-nvidia  does this install `libnvinfer.so` and plugins ?

Reply:
@peri044 yes, it does installed libnvinfer.so and plugins.
with apt-get install -y python3-libnvinfer-dev 
Here is the full list of packages that are being installed:
```
libnvinfer-dev libnvinfer-dispatch-dev libnvinfer-dispatch10 libnvinfer-headers-dev libnvinfer-headers-plugin-dev libnvinfer-lean-dev libnvinfer-lean10 libnvinfer-plugin-dev libnvinfer-plugin10
 libnvinfer-vc-plugin-dev libnvinfer-vc-plugin10 libnvinfer10 libnvonnxparsers-dev libnvonnxparsers10 python3-libnvinfer python3-libnvinfer-dev python3-libnvinfer-dispatch python3-libnvinfer-lean
```

Reply:
@peri044 
Thanks for your review, it is a very good point, if we do 
apt-get install -y python3-libnvinfer-dev 
we cannot ensure we use the exact 10.0.1 version, it might give us whatever the latest version.
In order to pin down the version, then I have to use something like
apt-get install -y libnvinfer10={TENSORRT_VERSION}.*  libnvinfer-plugin10={TENSORRT_VERSION}.* .......
it will need to specify the major version 10
Though I can do something like:
TENSORRT_MAJOR_VERSION=(echo ${TENSORRT_VERSION} | cut -d '.' -f 1)
apt-get install -y libnvinfer${TENSORRT_MAJOR_VERSION}={TENSORRT_VERSION}.* ......
@peri044  @narendasan   let me know if you are ok with this approach.

Reply:
> apt-get install -y libnvinfer${TENSORRT_MAJOR_VERSION}={TENSORRT_VERSION}.* ......
This looks good to me.